packages/react-meteor-data/react-meteor-data.d.ts

```diff
@@ -24,5 +26,12 @@ export function withTracker<TDataProps, TOwnProps>(options: {
 
 export function useSubscribe(name?: string, ...args: any[]): () => boolean;
 
+// Suspense
+export function useSubscribe(name: string, ...params: EJSON[]) : Meteor.SubscriptionHandle;
```

Comment:
It overloads the existing hook and thus returns a function or handle... It's incorrect, as it should be in a `/suspense` module, not in the top level (same goes for the new `useFind`).